js-watcher/routes.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
I really think it makes sense to have all functions operating on the same struct in a single file - could we merge them?

Reply:
I really want someone to take one quick look and see all the http routes that are exposed. Combining things in one file and removing `routes.go` file makes this harder.

I renamed `routes.go` to `server_routes.go` to indicate they are related.